### PR TITLE
Simple Edit to the Readme

### PR DESCRIPTION
**Pull request by Ulfingrad**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes
After discussing with Spencer for a bit, Smit and I decided to add a little bit more clarification to the Readme in order to make it a bit easier to follow (Specifically in the "Local Development" section). If there's anything wrong with what we've added (typos, grammar, etc.), please let us know and we'll get on it as soon as we can!

### CODE REVIEW

**Comment by SpencerKaiser** [comment]
Just a heads up for next time; you need to delete the space around the checkboxes in the pre-req section of the issue description:
```
- [x] Valid, checked checkbox
- [x ] Invalid checkbox; displays just as you see it
```

Reviewing the rest now!

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This app is still in the early stages of development and only some functionality
...
+Hangar uses Postgres 11, so you'll need to set it up on your machine. During the installation process, you will be prompted to create a password. You need to create PGUSER and PGPASSWORD in your `.env` file where PGUSER will be `postgres` and PGPASSWORD will be the password you enter upon installation. Create a database (we suggest `hangar`, if you chose something else or are running your server on a different port, make sure to create a `DATABASE_URL` value in `.env` with your override URL) in the PostgreSQL 11 server, ensure it is not a seperate server, and then run the app. When the app is deployed to a cloud environment, the `DATABASE_URL` `.env` var will be used (and is automatically set in Heroku when an associated service is connected to your app).
```
Comment (SpencerKaiser) [inline README.md:33]:
```suggestion
## Postgres
Hangar uses Postgres 11, so you'll need to set it up on your machine. If you don't have Postgres installed already, see the [installation](#installation) section below. 

Create a database (we suggest `hangar`, if you chose something else or are running your server on a different port, make sure to create a `DATABASE_URL` value in `.env` with your override URL) in the PostgreSQL 11 server (do not make a new server), and then run the app. If your Postgres server is protected by a password, you'll need to add `PGUSER` and `PGPASSWORD` to your `.env` file where `PGUSER` will be `postgres` and `PGPASSWORD` will be the Postgres server password entered during installation.

When the app is deployed to a cloud environment, the `DATABASE_URL` `.env` var will be used (and is automatically set in Heroku when an associated service is connected to your app).

### Installation
During the installation process (if you follow the steps on [postgresql.org](https://www.postgresql.org/download/)]), you will be prompted to create a password - make sure to use something you'll remember.
```

**Review by SpencerKaiser — CHANGES_REQUESTED** [review]
I think the other PR might be an update to this content (I'll look at that next) and I think @johnkahn already made this suggestion, but make sure to add the `PGUSER` and `PGPASSWORD` items to `.env.sample`. When you add them, make sure they're commented out with a note about when to use them since not all Postgres installations will require it.

**Comment by SpencerKaiser** [comment]
Finally, @Ulfingrad - looks like you committed directly to `main`; when you're done with this PR and it's been merged, you'll likely want to nuke your repo and re-clone.

File: README.md [hunk 2]
```diff
@@ -30,7 +30,15 @@ This app is still in the early stages of development and only some functionality
...
+## Postgres
+Hangar uses Postgres 11, so you'll need to set it up on your machine. If you don't have Postgres installed already, see the [installation](#installation) section below. 
+
+Create a database (we suggest `hangar`, if you chose something else or are running your server on a different port, make sure to create a `DATABASE_URL` value in `.env` with your override URL) in the PostgreSQL 11 server (do not make a new server), and then run the app. If your Postgres server is protected by a password, you'll need to add `PGUSER` and `PGPASSWORD` to your `.env` file where `PGUSER` will be `postgres` and `PGPASSWORD` will be the Postgres server password entered during installation.
+
+When the app is deployed to a cloud environment, the `DATABASE_URL` `.env` var will be used (and is automatically set in Heroku when an associated service is connected to your app).
+
+### Installation
+During the installation process (if you follow the steps on [postgresql.org](https://www.postgresql.org/download/)]), you will be prompted to create a password - make sure to use something you'll remember.
```
Comment (SpencerKaiser) [inline README.md:41]: Whoops I left a rogue bracket in there
```suggestion
During the installation process (if you follow the steps on [postgresql.org](https://www.postgresql.org/download/)), you will be prompted to create a password - make sure to use something you'll remember.
```

File: .env.sample [hunk 3]
```diff
@@ -6,4 +6,8 @@ SLACK_BOT_TOKEN="xoxb-XXXXXXXXXXXXXXXX"
 JUDGE_SECRET="secret"
 ADMIN_SECRET="something secretive"
 CHALLENGE_URL="https://your-sponsor-challenge.com"
-SLACK_NOTIFICATIONS_WEBHOOK_URL=""
\ No newline at end of file
+SLACK_NOTIFICATIONS_WEBHOOK_URL=""
+
+#Only uncomment and use these when downloading Postgres directly from postgresql.org/download/
```
Comment (SpencerKaiser) [inline .env.sample:11]:
```suggestion
#Only uncomment and use these if you have a custom user/pass for your Postgres Server
```

**Comment by Ulfingrad** [comment]
Also apologies for committing to main, I realized I did so after I sent the pull request, and was unsure on whether or not I should have just cancelled the pull request and tried again. I will definitely be sure to be more careful in the future!

**Review by SpencerKaiser — APPROVED** [review]
:shipit: 🚀 

@johnkahn any thoughts?

**Review by johnkahn — APPROVED** [review]

**Pull request merged**